packages/mdc-slider/foundation.js

```diff
@@ -339,7 +339,7 @@ class MDCSliderFoundation extends MDCFoundation {
 
   /**
    * Handles keydown events
-   * @param {Event} evt
+   * @param {?Event} evt
```

Comment:
This param should probably be non-null, because we call `evt.preventDefault()` below without checking if `evt` is non-null.

Reply:
Done